diff --git a/node_modules/@ephys/zod-to-ts/dist/utils.js b/node_modules/@ephys/zod-to-ts/dist/utils.js
index 06fc4a2..6d10224 100644
--- a/node_modules/@ephys/zod-to-ts/dist/utils.js
+++ b/node_modules/@ephys/zod-to-ts/dist/utils.js
@@ -97,7 +97,11 @@ export function addTsDocComment(node, text) {
     }
 }
 export function getSchemaDescription(schema, registry) {
-    return registry.get(schema)?.description;
+    let node = schema;
+    while (node.isOptional()) {
+        node = node.def.innerType;
+    }
+    return node.meta()?.description ? node.meta()?.description: registry.get(node)?.description;  
 }
 export function getSchemaIdentifier(schema, registry) {
     const id = (registry ?? globalRegistry).get(schema)?.id;
