diff --git a/node_modules/@types/presto-client/index.d.ts b/node_modules/@types/presto-client/index.d.ts
index b0f2f87..827d572 100644
--- a/node_modules/@types/presto-client/index.d.ts
+++ b/node_modules/@types/presto-client/index.d.ts
@@ -78,7 +78,7 @@ export interface ClientOptions {
     /**
      * Sets HTTP `Authorization` header with the provided string.
      */
-    custom_auth?: string;
+    custom_auth?: string | (() => Promise<string> | string);
     /**
      * Default catalog name
      */
