diff --git a/node_modules/agenda/dist/agenda/define.d.ts b/node_modules/agenda/dist/agenda/define.d.ts
index 0ccf6a1..f432450 100644
--- a/node_modules/agenda/dist/agenda/define.d.ts
+++ b/node_modules/agenda/dist/agenda/define.d.ts
@@ -31,6 +31,7 @@ export interface DefineOptions {
      */
     shouldSaveResult?: boolean;
 }
+// @ts-ignore
 export declare type Processor<T> = ((job: Job<T>) => Promise<void>) | ((job: Job<T>, done: () => void) => void);
 /**
  * Setup definition for job
