diff --git a/node_modules/snowflake-sdk/lib/http/base.js b/node_modules/snowflake-sdk/lib/http/base.js
index 626c313..224e8e9 100644
--- a/node_modules/snowflake-sdk/lib/http/base.js
+++ b/node_modules/snowflake-sdk/lib/http/base.js
@@ -232,6 +232,11 @@ function prepareRequestOptions(options) {
     requestOptions.httpAgent = agent;
   }
 
+  // Proxy is already being handled by Snowflake, tell Axios not to try proxying as well
+  if (this._connectionConfig.getProxy()) {
+    requestOptions.proxy = false;
+  }
+
   return requestOptions;
 }
 
