diff --git a/build/src/index.d.ts b/build/src/index.d.ts
index 4698e85f9cc9a964e9cc12de401454a6af5b231b..8872da4327ee0743d0fa07853adc7957a86ef8d3 100644
--- a/build/src/index.d.ts
+++ b/build/src/index.d.ts
@@ -19,6 +19,7 @@ import { AgentOptions as HttpAgentOptions } from 'http';
 import { PassThrough, Readable } from 'stream';
 import { TeenyStatistics } from './TeenyStatistics';
 export interface CoreOptions {
+    followRedirect?: boolean;
     method?: string;
     timeout?: number;
     gzip?: boolean;
diff --git a/build/src/index.js b/build/src/index.js
index a101736025f90dc036155e99c0e0b432291e0bcb..acc7769eb0ffd927c728dcce19caa17f218bc49b 100644
--- a/build/src/index.js
+++ b/build/src/index.js
@@ -37,6 +37,7 @@ exports.RequestError = RequestError;
 function requestToFetchOptions(reqOpts) {
     const options = {
         method: reqOpts.method || 'GET',
+        ...(reqOpts.followRedirect === false && { redirect: 'error' }),
         ...(reqOpts.timeout && { timeout: reqOpts.timeout }),
         ...(typeof reqOpts.gzip === 'boolean' && { compress: reqOpts.gzip }),
     };
